src/sentry/runner/commands/createuser.py

```diff
@@ -0,0 +1,96 @@
+"""
+sentry.runner.commands.createuser
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2015 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import, print_function
+
+import click
+from sentry.runner.decorators import configuration
+
+
+def _get_field(field_name):
+    from sentry.models import User
+    return User._meta.get_field(field_name)
+
+
+def _get_email():
+    from django.core.exceptions import ValidationError
+    rv = click.prompt('Email')
+    field = _get_field('email')
+    try:
+        return field.clean(rv, None)
+    except ValidationError as e:
+        raise click.ClickException('; '.join(e.messages))
+
+
+def _get_password():
+    from django.core.exceptions import ValidationError
+    rv = click.prompt('Password', hide_input=True, confirmation_prompt=True)
+    field = _get_field('password')
+    try:
+        return field.clean(rv, None)
+    except ValidationError as e:
+        raise click.ClickException('; '.join(e.messages))
+
+
+def _get_superuser():
+    return click.confirm('Should this user be a superuser?', default=False)
+
+
+@click.command()
+@click.option('--email')
+@click.option('--password')
+@click.option('--superuser/--no-superuser', default=None, is_flag=True)
+@click.option('--no-password', default=False, is_flag=True)
+@click.option('--no-input', default=False, is_flag=True)
+@configuration
+def createuser(email, password, superuser, no_password, no_input):
+    "Create a new user."
+    if not no_input:
```

Comment:
would you mind making a few basic tests that just say "does this command do anything at all" just so the test suite hits it